sdk/python/feast/feature_service.py

```diff
@@ -14,16 +16,31 @@
 
 
 class FeatureService:
+    """
+    A FeatureService object is used to group features together for retrieval.
+    The features may all be part of the same Feature view, or from a combination of Feature views.
+
+    """
+
     name: str
     features: List[FeatureViewProjection]
+    tags: Dict[str, str]
     created_timestamp: Optional[datetime] = None
     last_updated_timestamp: Optional[datetime] = None
 
     def __init__(
         self,
         name: str,
         features: List[Union[FeatureTable, FeatureView, FeatureViewProjection]],
+        tags: Optional[Dict[str, str]] = None,
     ):
+        """
+        Create a new Feature Service object.
```

Comment:
This isn't a Google style docstring.

Also, we should either use the class name (when talking about the class itself) or just lower case naming like `feature service` when describing the concept.